src/components/CallFeedback/CallFeedback.tsx

```diff
@@ -1,282 +1,157 @@
-import React, { useState } from "react";
+import React from "react";
 import { useTranslation } from "react-i18next";
 
-import { Layout, Rate, Button, Space, Card, Radio } from "antd";
+import { Layout, Button } from "antd";
 import { getLottieOptions } from "src/utils/UI";
 import Lottie from "react-lottie";
 import GoodbyeLottie from "src/assets/Lottie/Goodbye.json";
 import TeamWaveLottie from "src/assets/Lottie/TeamWave.json";
 import CyclingLottie from "src/assets/Lottie/GirlCycling.json";
 import ShrugLottie from "src/assets/Lottie/Shrug.json";
 import RocketLottie from "src/assets/Lottie/Rocket.json";
-import { getContactsFirstNames, showToast } from "src/utils";
-import { Call } from "src/types/Meeting";
-import { PopupButton } from "@typeform/embed-react";
+import FamilyLottie from "src/assets/Lottie/Family.json";
 
-export type CallFeedbackType =
-  | "natural_expiration"
-  | "deliberate"
-  | "early_exit"
-  | "terminated"
-  | "intake_form";
+import { getContactsFirstNames, getRandomItem } from "src/utils";
+import { Call } from "src/types/Meeting";
+import CallRating from "./CallRating";
+import CallFeedbackIntakeForm from "./CallFeedbackIntakeForm";
+import { useAppNavigate } from "src/hooks/useAppNavigate";
+import { H2 } from "../Typography";
+
+/*
+  No show: other participant didn't join at all
+  Early exit: the incarcerated user left the call
+  Ended: 'call_status' socket event. Call naturally expired.
+  Terminated: 'call_status' socket event. DOC terminated the call.
+*/
+export type CallExitType = "no_show" | "early_exit" | "ended" | "terminated";
 
 interface Props {
   call: Call;
-  navigate: (path: string) => void;
   rateCall: (rating: number, feedback?: string) => void;
-  type: CallFeedbackType;
+  type: CallExitType;
   logout: () => void;
-  markIntakeFormAsComplete: () => void;
+  onSubmitIntakeForm: () => void;
+  promptIntakeForm: boolean;
 }
 
+const FeedbackWrapper = ({
+  children,
+}: {
+  children: React.ReactNode[] | React.ReactNode;
+}) => (
```

Comment:
the `React.FC` type will take care of declaring children if you like:

```suggestion
const FeedbackWrapper: React.FC = ({children}) => (
```